scrapylib/hcf.py

```diff
@@ -211,12 +223,6 @@ def _get_new_requests(self):
         self._msg('Read %d new batches from slot(%s)' % (num_batches, self.hs_consume_from_slot))
         self._msg('Read %d new links from slot(%s)' % (num_links, self.hs_consume_from_slot))
 
-    def _save_new_links_count(self):
-        """ Save the new extracted links into the HCF."""
-        for slot, new_links in self.new_links.items():
-            self._msg('Stored %d new links in slot(%s)' % (len(new_links), slot))
```

Comment:
The purpose of storing the counts myself (instead of `len`) is to avoid storing 100 million strings in `new_links` in my case. Now with `hcf_dont_filter` set to `True` in `request.meta`, `new_links` could be empty and space saved. :)